Gopkg.lock

```diff
@@ -29,6 +29,18 @@
   ]
   revision = "87d4990451a858cc210399285be976e63bc3c364"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/globalsign/mgo"
```

Comment:
Just to be clear, this is still a breaking change and will need to wait to be merged into `master` until that time.

Reply:
This branch targets dev :D

Reply:
I know :) I just want to make sure that anybody looking through the history sees that we were aware of the breaking changes.